lib/sda/tree.go

```diff
@@ -258,7 +259,7 @@ func (tm TreeMarshal) MakeTree(il *EntityList) (*Tree, error) {
 
 // MakeTreeFromList creates a sub-tree given an EntityList
 func (tm *TreeMarshal) MakeTreeFromList(parent *TreeNode, il *EntityList) *TreeNode {
-	idx, ent := il.Search(tm.EntityId)
+	idx, ent := il.SearchEntity(tm.EntityId)
```

Comment:
`SignSchnorr` is a 'static' function from a packet, whereas `SearchEntity` is a function defined on a structure. As such I wouldn't treat them the same way for naming. Of course `entity.SearchEntity` is redundant, but if I have an entityList `il`, saying `il.SearchEntity` is less redundant. Just a bit...